src/app/modules/familymigration/familymigrationFeedback.js

```diff
@@ -14,60 +14,97 @@ fm.directive('fmFeedback', ['IOService', function (IOService) {
       return function ($scope, element, attrs) {
         $scope.showForm = true
         $scope.showThanks = false
-        $scope.feedback = { whynot: {} }
+        $scope.feedback = {
+          reasonForNotMatch: {}
+        }
 
         var options
         if ($scope.passed) {
-          options = [
-            { value: 'failed-a-salaried', label: 'Not Passed on Cat A Salaried' },
-            { value: 'failed-b-nonsalaried', label: 'Not Passed on Cat B Non-Salaried' },
-            { value: 'failed-f', label: 'Not Passed on Cat F Self Assessment (1 Year)' },
-            { value: 'failed-g', label: 'Not Passed on Cat G Self Assessment (2 Years)' }
+          options = [{
+              value: 'failed-a-salaried',
+              label: 'Not Passed on Cat A Salaried'
+            },
+            {
+              value: 'failed-b-nonsalaried',
+              label: 'Not Passed on Cat B Non-Salaried'
+            },
+            {
+              value: 'failed-f',
+              label: 'Not Passed on Cat F Self Assessment (1 Year)'
+            },
+            {
+              value: 'failed-g',
+              label: 'Not Passed on Cat G Self Assessment (2 Years)'
+            }
           ]
         } else {
-          options = [
-            { value: 'passed-a-salaried', label: 'Passed on Cat A Salaried' },
-            { value: 'passed-b-nonsalaried', label: 'Passed on Cat B Non-Salaried' },
-            { value: 'passed-f', label: 'Passed on Cat F Self Assessment (1 Year)' },
-            { value: 'passed-g', label: 'Passed on Cat G Self Assessment (2 Years)' }
+          options = [{
+              value: 'passed-a-salaried',
+              label: 'Passed on Cat A Salaried'
+            },
+            {
+              value: 'passed-b-nonsalaried',
+              label: 'Passed on Cat B Non-Salaried'
+            },
+            {
+              value: 'passed-f',
+              label: 'Passed on Cat F Self Assessment (1 Year)'
+            },
+            {
+              value: 'passed-g',
+              label: 'Passed on Cat G Self Assessment (2 Years)'
+            }
           ]
         }
 
         $scope.conf = {
           match: {
             label: 'Did IPS match the paper assessment?',
             inline: true,
-            options: [{ value: 'yes', label: 'Yes' }, { value: 'no', label: 'No' }],
-            onClick: function (opt, scope) {
-              setFeedbackVisibility(opt.value)
+            options: [{
+              value: 'yes',
+              label: 'Yes'
+            }, {
+              value: 'no',
+              label: 'No'
+            }],
+            onClick: function (options) {
+              setFeedbackVisibility(options.value)
             }
           },
-          whynot: {
-            id: 'whynot',
+          reasonForNotMatch: {
+            id: 'reasonForNotMatch',
```

Comment:
This could be the root of the test errors. I suggest updating the BDD to match your new ID and see if that works.